src/cmakeFindModules/OpenMVGConfig.cmake.in

```diff
@@ -82,6 +82,19 @@ endif (NOT EXISTS ${CURRENT_ROOT_INSTALL_DIR}/include/openMVG/version.hpp)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+# Record state of build time dependencies
+set(OpenMVG_USE_INTERNAL_CEREAL "@OpenMVG_USE_INTERNAL_CEREAL@")
+set(OpenMVG_USE_OPENMP "@OpenMVG_USE_OPENMP@")
+
+# Find transitive dependencies if required
+include(CMakeFindDependencyMacro)
+if (OpenMVG_USE_OPENMP)
+  find_dependency(OpenMP REQUIRED)
+endif()
+if (NOT OpenMVG_USE_INTERNAL_CEREAL)
+  find_dependency(cereal REQUIRED)
+endif()
+
```

Comment:
I would add the following too since it is used as a component too by OpenMVG
-> do a search for `::` in any *.txt file in openMVG/src and you will see it

```
set(THREADS_PREFER_PTHREAD_FLAG ON)
find_package(Threads REQUIRED)
```